torch/fx/graph.py

```diff
@@ -148,8 +152,14 @@ def node_copy(self, node: Node, arg_transform: Callable[[Node], Argument] = lamb
         return self.create_node(node.op, node.target, args, kwargs, name)
 
     def output(self, result: Argument):
-        self.result = result
-        self._mark_uses(result)
+        assert len(self._nodes) != 0
+        assert self._nodes[-1].op == 'return'
+        self._nodes[-1].args = result
+
+    @property
```

Comment:
`result` is redundant now with the presence of a return node. The code in the if statement handling nodes should be the thing handling the result otherwise you end up having two places where everything handles stuff.